spaces/models.py

```diff
@@ -37,6 +37,26 @@ def __str__(self):
         return "{} ({})".format(self.name, self.parent.name if self.parent else "")
 
 
+class ServiceSector(models.Model):
+    """
+    Model representation of Service Sector that contains and manages
+    units and application periods.
+    """
+
+    name = models.CharField(verbose_name=_("Name"), max_length=255)
+    units = models.ManyToManyField(
+        "Unit", verbose_name=_("Units"), related_name="departments"
```

Comment:
related name seems to be wrong/inconsistent

Reply:
Fixe'd